site/docs/components/card/accessibility.mdx

```diff
@@ -8,6 +8,17 @@ data:
   $ref: ./#/data
 ---
 
+In adherence to accessibility standards, the boundaries of a user interface component are only subjective to the 3:1 contrast minimum if they serve to identify the components purpose.
+
+Salt Card components incorporate subtle borders as part of their design. To ensure these Cards meet accessibility criteria, it’s imperative to check that any content (i.e., text or icon) used within them maintain a minimum contrast ratio of 4:5:1 against the Card background. This can be achieved by using the Salt --content tokens.
```

Comment:
When referencing tokens we should make them align to code e.g. --salt-content-* or something similar